ui/ui-components/pages/profile/[id]/edit.tsx

```diff
@@ -411,17 +415,19 @@ export default function Page(props) {
                   </Form.Control>
                 </Form.Group>
               </Col>
-              <Col md={3}>
-                <div style={{ paddingTop: 34 }} />
-                <LoadingButton
-                  variant="outline-primary"
-                  size="sm"
-                  type="submit"
-                  disabled={loading}
-                >
-                  Add
-                </LoadingButton>
-              </Col>
+              {process.env.GROUPAROO_UI_EDITION === "config" ? null : (
```

Comment:
I think above (`<Col md={9}>`) will need to be updated to be `md=12` since there will only be one column to fill the whole area

Reply:
I think we actually want the whole row gone.  The col-9 section is a list of possible groups... which doesn't matter if you can't add them.  Got that updated!